### PR TITLE
Update validations for login / sign up. Add proper checks for constraints we enforce

### PR DESCRIPTION
**Pull request by witehound**

### CODE REVIEW

File: server/src/middleware/validators/userRoutesValidator.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const userSignUp = celebrate({
     name: Joi.string().required(),
     email: Joi.string().required(),
     password: Joi.string().required(),
```
Comment (andrew-bierman) [inline server/src/middleware/validators/userRoutesValidator.ts:20]: We have constraints we should be validation here also, such as a couple password things

**Review by andrew-bierman — CHANGES_REQUESTED** [review]

File: server/src/middleware/validators/userRoutesValidator.ts (hunk 1 above)
Comment (witehound) [reply server/src/middleware/validators/userRoutesValidator.ts:20]: Like what?

Comment (andrew-bierman) [reply server/src/middleware/validators/userRoutesValidator.ts:20]: See the userModel we have a few requirements

Comment (andrew-bierman) [reply server/src/middleware/validators/userRoutesValidator.ts:20]: @witehound this is still pending

Comment (witehound) [reply server/src/middleware/validators/userRoutesValidator.ts:20]: Those are all the required fields! And now the function works as it should really!

Comment (witehound) [reply server/src/middleware/validators/userRoutesValidator.ts:20]: @andrew-bierman just checked and I think this issue has been properly resolved

Comment (andrew-bierman) [reply server/src/middleware/validators/userRoutesValidator.ts]: Are there any changes in this PR that are not already in andrew_testing ? After syncing branch it does not show any touched files

Comment (witehound) [reply server/src/middleware/validators/userRoutesValidator.ts]: Oh Yh when we had the zod validation issue the last branch that was working was this branch so I reverted to this branch and started building till you finally updated andrew_testing? So most of the lastest brabches I have already have this same changed !

**Comment by andrew-bierman** [comment]
Closing because nothing seems to be changing anymore. Note to self, add user validations similar to model ie password

**Pull request closed without merging**